Reporting_SSIS/Reporting_SSIS/AdventureWorks2016CTP3.conmgr

```diff
@@ -3,6 +3,10 @@
   DTS:ObjectName="AdventureWorks2016CTP3"
   DTS:DTSID="{65FD546B-5455-42C0-813A-B3618EF489CE}"
   DTS:CreationName="OLEDB">
+  <DTS:PropertyExpression
+    DTS:Name="InitialCatalog">@[$Project::SourceDBName]</DTS:PropertyExpression>
+  <DTS:PropertyExpression
+    DTS:Name="ServerName">@[$Project::ServerName]</DTS:PropertyExpression>
```

Comment:
@Sreenitha-Sampath As we have source and destination servers, rename ServerName to SourceServerName

Reply:
Both the source and destination servers are the same so I think its better to leave that as server name